tools/submission/submission-checker.py

```diff
@@ -0,0 +1,554 @@
+    path = os.path.join(*path)
+    return [f for f in os.listdir(path) if os.path.isdir(os.path.join(path, f))]
+
+
+def list_files(*path):
+    path = os.path.join(*path)
+    return [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+
+
+def split_path(m):
+    return m.replace("\\", "/").split("/")
+
+
+def ignore_errors(line):
+    if "check for ERROR in detailed" in line:
+        return True
+    if "Loadgen built with uncommitted changes" in line:
+        return True
+    if "Ran out of generated queries to issue before the minimum query count and test duration were reached" in line:
+        return True
+    if "CAS failed":
+        return True
+    return False
+
+
+def check_accuracy_dir(config, model, dir):
+    is_valid = False
+    acc = None
+    model_norm = model_map(config, model)
+    acc_type, acc_target = config.get_accuracy_target(model_norm)
+    if not isinstance(acc_target, list):
+        acc_target = [acc_target]
+    acc_target = list(sorted(acc_target, reverse=True))
+    pattern = ACC_PATTERN[acc_type]
+    with open(os.path.join(dir, "accuracy.txt"), "r") as f:
+        for line in f:
+            m = re.match(pattern, line)
+            if m:
+                is_valid = True
+                acc = m.group(1)
+                break
+
+    if acc:
+        for a in acc_target:
+            if float(acc) >= a:
+                is_valid = True
+                break
+        if not is_valid:
+            log.error("%s accuracy not met: expected=%f, found=%f", dir, acc_target, acc)
+
+    # check if there are any errors in the detailed log
+    fname = os.path.join(dir, "mlperf_log_detail.txt")
+    if not os.path.exists(fname):
+        log.error("%s is missing", fname)
+        is_valid = False
+    else:
+        with open(fname, "r") as f:
+            for line in f:
+                # look for: ERROR
+                if "ERROR" in line:
+                    if ignore_errors(line):
+                        continue
+                    # TODO: should this be a failed run?
+                    log.error("%s contains error: %s", fname, line)
+                    is_valid = False
+    return is_valid, acc
+
+
+def check_performance_dir(config, model, dir):
+    is_valid = False
+    rt = {}
+    # look for: Result is: VALID
+    fname = os.path.join(dir, "mlperf_log_summary.txt")
+    with open(fname, "r") as f:
+        for line in f:
+            m = re.match(r"^Result\s+is\s*\:\s+VALID", line)
+            if m:
+                is_valid = True
+            m = re.match(r"^\s*([\w\s.\(\)\/]+)\s*\:\s*([\w\+\.]+).*", line)
+            if m:
+                rt[m.group(1).strip()] = m.group(2).strip()
+
+    model = model_map(config, model)
+    performance_sample_count = config.get_performance_sample_count(model)
+    if int(rt['performance_sample_count']) < performance_sample_count:
+        log.error("%s performance_sample_count should be %d", fname, performance_sample_count)
```

Comment:
`should be %d` -> `should be at least %d`

Reply:
fixed.